### DISCUSSION TITLE
filament shiled create role show error

### DISCUSSION THREAD

**Discussion by ayemin-tun** [filamentphp/filament#11401] [General]
In my website when I install filament shield and when I go to filament role create it show like this

![image](https://github.com/filamentphp/filament/assets/76741291/a33bf161-86a1-4d15-8b9c-49e87327c744)

I don't know why  please advice me how to fix this

**Comment by leandrocfe** [comment]
Please ask here: https://discord.com/channels/883083792112300104/927502073543675976
